src/annotator/components/test/ToastMessages-test.js

```diff
@@ -25,14 +25,14 @@ describe('ToastMessages', () => {
     const wrapper = createComponent();
 
     // Initially messages is empty
-    assert.lengthOf(wrapper.find('BaseToastMessages').prop('messages'), 0);
+    assert.lengthOf(wrapper.find('[messages]').prop('messages'), 0);
```

Comment:
There's a couple of these changes in some tests. The reason is that the component is called `ToastMessages` instead of `BaseToastMessages`. That's the name it will probably have when moved to frontend-shared.

However, it is here imported as a default export with the name `BaseToastMessages`, because it is being imported in a component called `ToastMessages` as well.

That makes `wrapper.find('BaseToastMessages')` not work, but `wrapper.find('ToastMessages')` does not work either, because it resolves the parent component.

I thought finding by the property we actually want to evaluate was a pragmatic and simple solution for now.

In future, when we can import it like `import { ToastMessages as BaseToastMessages } from `@hypothesis/frontend-shared'`, I expect the previous approach to work again, and we can get it back.

Reply:
This logic makes sense, but I'd suggest to wrap this up in a helper (eg. `getInnerToastMessages`).